### DISCUSSION TITLE
The importance of translation in Open-Source World!

### DISCUSSION THREAD

**Discussion by petrichor-494** [elementary/edw#46] [Ideas]
Hello, 

I would like to give a talk regarding the topic, but unfortunately, I am run out of time and cannot prepare it. Thus, I would like to hear someone talk about it or a similar topic.

Best wishes,
Aga from Kurdistan <3

**Comment by JeysonFlores** [comment]
The EDW will be in two months. Are you really sure you cannot give the talk?

**Reply by petrichor-494** [reply]
yes, coz I am busy with working and studying 24/7 :(
